dataland-frontend/src/components/resources/frameworkDataSearch/euTaxonomy/configMLDT/configForEutaxonomyFinancialsMLDT.ts

```diff
@@ -71,42 +82,43 @@ export const configForEuTaxonomyFinancialsMLDT = [
         type: "cell",
         label: euTaxonomyKpiNameMappings.fiscalYearDeviation,
         explanation: euTaxonomyKpiInfoMappings.fiscalYearDeviation,
-        shouldDisplay: (): boolean => true,
+        shouldDisplay: (dataset: EuTaxonomyDataForFinancials): boolean => !!dataset.fiscalYearDeviation,
         valueGetter: plainStringValueGetterFactory("fiscalYearDeviation"),
       },
       {
         type: "cell",
         label: euTaxonomyKpiNameMappings.fiscalYearEnd,
         explanation: euTaxonomyKpiInfoMappings.fiscalYearEnd,
-        shouldDisplay: (): boolean => true,
+        shouldDisplay: (dataset: EuTaxonomyDataForFinancials): boolean => !!dataset.fiscalYearEnd,
```

Comment:
why not use setShowIf here as well? why make a difference for exactly these values?

Reply:
done

Reply:
both functions we have won't fit here, because value is a string and neither a YesNo or a Datapoint. And writing a function for three occurances in a config that is going to be discharged in a short amount of time seems unnecessary

Reply:
Could be templated. But valid argument.

Reply:
Implemented